sadu-updater/src/main/java/de/chojo/sadu/updater/SqlUpdater.java

```diff
@@ -171,6 +171,10 @@ public void init() throws IOException, SQLException {
             return;
         }
 
+        if(sqlVersion.major() > version.major() || sqlVersion.patch() > version.patch()){
```

Comment:
This will not work, imagine the latest version is 2.0. And the database has 1.3. Now the database's patch version is higher than the latest and this check will throw an exception.